### DISCUSSION TITLE
Accessing onPremisesExtensionAttributes via graph.microsoft.com

### DISCUSSION THREAD

**Discussion by JazzHoolahawk** [microsoft-search/pnp-modern-search#3382] [Q&A] [answered]
Hi all,

We want to include some Custom Extension Attributes data using Ms Graph for search results when searching on a persons name. For example, if you run the following query https://graph.microsoft.com/v1.0/me?$select=onPremisesExtensionAttributes via the https://developer.microsoft.com/en-us/graph/graph-explorer

Is it possible to get access to this the the results of the api query in the modern search results out of the box or does this require any additional development on our part.

**Comment by kasperbolarsen** [comment, marked as answer]
Hi,
At the moment Graph People search is still in beta, and lacking. So the only viable approach is to sync the custom attributes to User Profile properties and use SharePoint localpeopleresults as your data source. 
We all expect that to change,  but the roadmap from MS hasn't provided an ETA yet.

**Reply by JazzHoolahawk** [reply]
Hi Kasperbolarsen, 
Thanks for your reply it is really appreciated.

That makes sense. We are hoping to avoid having to sync the custom attributes to the UPS. I keep my eye on the roadmap for a future announcement :)

Cheers
David
